doc-surrealdb_versioned_docs/version-latest/querying/graphql/http.mdx

```diff
@@ -0,0 +1,121 @@
+---
+sidebar_position: 2
+sidebar_label: GraphQL via HTTP 
+title: GraphQL via HTTP | GraphQL
+description: In this section, you will explore querying SurrealDB using the GraphQL HTTP endpoint. The HTTP API is designed to be simple and intuitive, with any interface that provides a consistent way to interact with the database.
+---
+
+import Since from '@site/src/components/Since';
+
+# GraphQL via HTTP
+
+SurrealDB provides a powerful HTTP API that allows you to interact with the database programmatically. This API can be used to perform a wide range of database operations, from querying data to modifying records and managing database structures.
+
+The HTTP API is designed to be simple and intuitive, with a RESTful interface that provides a consistent way to interact with the database. You can use the API to perform a wide range of database operations, from querying data to modifying records and managing database structures.
+
+## Starting a new connection 
+
+Before you can start making queries, you need to start SurrealDB with the GraphQL module enabled. You can do this by setting the `SURREAL_EXPERIMENTAL_GRAPHQL` [environment variable](/docs/surrealdb/installation/running/env_vars) to `true` and starting a new instance of SurrealDB with the [`surreal start` command](/docs/surrealdb/cli/start).
+
+```bash
+SURREAL_EXPERIMENTAL_GRAPHQL=true surreal start --log debug --user root --password root
+```
+
+:::note
+__Note:__ By running SurrealDB with the GraphQL module enabled, you are opting into an experimental feature. While the GraphQL module is fully functional, it is still considered experimental and may not be as stable as the core SurrealQL module. We welcome your feedback and contributions to help improve the feature and make it more robust. 
+
+GraphQL is a pre-release feature with known security flaws. This is not recommended for production use
```

Comment:
I wonder if we can take out this line? The note above I think makes it pretty clear that it is experimental which includes the domain of security IMO. Or maybe combine them? As a reader in the preview it feels like the moment you finish reading the warning there is suddenly another one which is a bit jarring.